dask/sizeof.py

```diff
@@ -84,6 +84,13 @@ def sizeof_pandas_index(i):
             p += object_size(i)
         return int(p) + 1000
 
+    @sizeof.register(pd.MultiIndex)
+    def sizeof_pandas_multiindex(i):
+        p = int(sum(object_size(l) for l in i.levels))
+        for c in i.codes if hasattr(i, "codes") else i.labels:
+            p += c.nbytes
+        return int(p)
```

Comment:
@mrocklin I think removing the `+ 1000` here fixes the test on 3.5, but since it's present for all the pandas dtypes I assume there's a reason for it? If so I can put it back and change the test to account for the +1000 offset

Reply:
I suspect that this was just accounting for the size of the pandas data structure itself, roughly.

Yeah, it looks like, at least the serialized cost is roughly this

```python
In [1]: import pandas as pd

In [2]: import pickle

In [3]: len(pickle.dumps(pd.DataFrame()))
Out[3]: 533

```

Reply:
I would change the test somewhat

Reply:
done